src/main/java/io/cryostat/recordings/RecordingsModule.java

```diff
@@ -148,4 +160,29 @@ static ScheduledExecutorService provideNotificationScheduler() {
         Runtime.getRuntime().addShutdownHook(new Thread(ses::shutdown));
         return ses;
     }
+
+    @Provides
+    @Singleton
+    static RecordingMetadataManager provideRecordingMetadataManager(
+            @Named(ConfigurationModule.CONFIGURATION_PATH) Path confDir,
```

Comment:
I'm not sure if the `CONFIGURATION_PATH` is the best parent for this purpose, but since we eventually want to move this to be backed by a proper database of some form rather than just manually managing JSON files on disk, I think it's okay for now. Please just leave a comment here that this is a `FIXME` and the manager should either be updated to use a database connection, or a new filesystem path should be created and used for this kind of purpose instead.